.github/actions/run_docker/entrypoint.sh

```diff
@@ -47,7 +49,10 @@ test_targets()
   return 0;
 }
 
-set -e
+# Temporary fix for ARM runtime
+rm /lib/ld-linux-armhf.so.3
+ln -s /usr/arm-linux-gnueabihf/lib/ld-linux-armhf.so.3 /lib/ld-linux-armhf.so.3
```

Comment:
What is happening and how is WASM related to arm runtime?

Reply:
I had to move to the v4 of our Docker. It appears to have a bug on how the ARM 32bits works now and need to fix it.
To not have spurrious fail, I added that. This is totally temporary.